prime-router/src/test/kotlin/common/TestcontainersUtils.kt

```diff
@@ -15,12 +15,6 @@ object TestcontainersUtils {
         customImageName: String,
         customEnv: Map<String, String> = emptyMap(),
     ): GenericContainer<*> {
-        val containerKey = Pair(customImageName, customEnv)
```

Comment:
The caching here was not safe across multiple gradle test runners and can result in random tests beginning to fail even if they were previously passing.